content/en/developers/marketplace/offering.md

```diff
@@ -0,0 +1,262 @@
+| Type               | `.jpg` or `.png`.                                                                                                                                 |
+| Size               | The average is around 500KB. The maximum image size is 1MB.                                                                                       |
+| Dimensions         | The aspect ratio must be 16:9 exactly and fit these specifications:<br><br>- Width: 1440px<br>- Minimum height: 810px<br>- Maximum height: 2560px |
+| Name               | Use letters, numbers, underscores, and hyphens. Spaces are not allowed.                                                                           |
+| Color Mode         | RGB                                                                                                                                               |
+| Color Profile      | sRGB                                                                                                                                              |
+| Description        | The maximum number of characters allowed is 300.                                                                                                  |
+
+Follow this template to define the `media` object in the media carousel which includes an image, a video thumbnail, and a video:
+
+{{< code-block lang="json" filename="manifest.json" collapsible="true" >}}
+"media": [
+      {
+        "media_type": "image",
+        "caption": "A Datadog Marketplace Integration OOTB Dashboard",
+        "image_url": "images/integration_name_image_name.png"
+      },
+      {
+        "media_type": "video",
+        "caption": "A Datadog Marketplace Integration Overview Video",
+        "image_url": "images/integration_name_video_thumbnail.png",
+        "vimeo_id": 123456789
+      },
+    ],
+{{< /code-block >}}
+
+For more information, see [Integrations Assets Reference][19].
+
+### Open a pull request
+
+Open a pull request that contains your integration tile's asset files (such as images, not videos) in the `marketplace` repository. Automatic tests run checks in Azure DevOps pipelines to verify that your pull request is in good shape and contains all the required content to be updated.
+
+To request access to the Azure DevOps pipeline, leave a comment in the pull request requesting access.
+
+### Obtain approval to merge
+
+Once your pull request passes all the checks, reviewers from the `Datadog/agent-integrations`, `Datadog/marketplace-review`, and `Datadog/documentation` teams provide suggestions and feedback on best practices. 
+
+Once you have addressed the feedback and re-requested reviews, these reviewers approve your pull request and the integration tile is enabled in your sandbox account. This allows you to validate and preview additional changes in the integation tile on the Datadog Marketplace before your pull request is merged. 
```

Comment:
```suggestion
Once you have addressed the feedback and re-requested reviews, these reviewers approve your pull request. Please let the Marketplace team know if you'd like to first preview the integration tile in your sandbox account. This allows you to validate and preview additional changes in the integration tile on the Datadog Marketplace before your pull request is merged. 
```
We typically ask if they'd like to preview it first. We don't always automatically do it for each partner as some like to just publish it.